src/components/map/sectors/MapSectorsList.vue

```diff
@@ -1,36 +1,141 @@
 <template>
-    <map-sector
-        v-for="(sector, index) in firs"
-        :key="sector.fir.feature.id as string + index"
-        :atc="sector.atc"
-        :fir="sector.fir"
-    />
+    <template v-if="store.localSettings.traffic?.vatglassesLevel === false">
```

Comment:
Let's add global computed or store getter for is vatglasses enabled or not

Reply:
The needed values are true/false/number. Can you give me an example where you want that setting?